### PR TITLE
Habit facts admin 1765

### PR DESCRIPTION
**Pull request by IvanBegar**
https://github.com/ita-social-projects/GreenCity/issues/1765

### CODE REVIEW

**Review by forestik — APPROVED** [review]

File: core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+import springfox.documentation.annotations.ApiIgnore;
+import javax.validation.Valid;
+import java.util.List;
+
+@Controller
```
Comment (vterlyha) [inline core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java:29]: Change to RestCOntroller annotation - and won't be no need in adding @ResponseBody to methods

File: core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+     *
+     * @return {@link HabitFactVO} instance.
+     * @author Ivan Behar
+     */
+    @ApiOperation(value = "Get habit facts by id.")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = HabitFactDtoResponse.class),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+    })
+    @GetMapping("/find")
```
Comment (vterlyha) [inline core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java:51]: {id} - id should be passed as Path param

File: core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+     * @return {@link GenericResponseDto} with of operation and errors fields
+     * @author Ivan Behar
+     */
+    @ApiOperation(value = "Save habit facts")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = GenericResponseDto.class),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+    })
+    @ResponseBody
+    @PostMapping("/")
```
Comment (vterlyha) [inline core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java:89]: Should be empty

Comment (IvanBegar) [reply core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java]: Fixed

File: core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java (hunk 2 above)
Comment (IvanBegar) [reply core/src/main/java/greencity/webcontroller/ManagementHabitFactsController.java]: fixed

**Review by KhonkoDmytro — APPROVED** [review]

**Review by Markiyan000 — APPROVED** [review]

**Pull request merged**